src/test/java/uk/ac/ebi/eva/test/utils/DropStudyJobTestUtils.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2015-2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.test.utils;
+
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBCollection;
+
+import static org.junit.Assert.assertEquals;
+import static uk.ac.ebi.eva.commons.models.converters.data.VariantToDBObjectConverter.FILES_FIELD;
+import static uk.ac.ebi.eva.commons.models.converters.data.VariantToDBObjectConverter.STATS_FIELD;
+import static uk.ac.ebi.eva.commons.models.data.VariantSourceEntity.STUDYID_FIELD;
+
+public class DropStudyJobTestUtils {
```

Comment:
I would say that the methods improve code clarity and reduce redundancy, but the main reason to extract them was in fact to highlight the common checks between the job and the steps.